shell/platform/windows/settings_plugin.h

```diff
@@ -24,36 +27,42 @@ class SettingsPlugin {
 
   virtual ~SettingsPlugin();
 
-  static std::unique_ptr<SettingsPlugin> Create(BinaryMessenger* messenger,
-                                                TaskRunner* task_runner);
-
   // Sends settings (e.g., platform brightness) to the engine.
   void SendSettings();
 
   // Start watching settings changes and notify the engine of the update.
-  virtual void StartWatching() = 0;
+  virtual void StartWatching();
 
   // Stop watching settings change. The `SettingsPlugin` destructor will call
   // this automatically.
-  virtual void StopWatching() = 0;
+  virtual void StopWatching();
 
  protected:
```

Comment:
These remain protected rather than being made private since they're overridden by the mock and used in the tests in settings_plugin_unittests.cc.